foss4g2022.html

```diff
@@ -250,6 +276,201 @@ <h2>All-in-one software suite</h2>
         <img src="img/hexagons_3d_white_outlier.png" class="stretch">
 </section>
 
+
+<section>
+    <h2>Integration with Jupyter Notebooks</h2>
+    <ul>
+        <li>Notebooks: Mix of text, code, results, images, &hellip;</li>
+        <li>Usage: Prototyping, reproducibility, tutorials, &hellip;</li>
+        <li>Now in GRASS GIS: Python functions for easy integration</li>
+    </ul>
+    <img src="img/jupyter_text_images_3d.png" class="stretch">
+    <p>by Caitlin Haedrich</p>
+</section>
+
+
+<section>
+    <h2>Integration with Jupyter Notebooks</h2>
+    <p>
+        Web map with base maps, zooming, customizations, &hellip;
+    </p>
+    <img src="img/jupyter_interactive_viewshed.png" class="stretch">
+    <p>
+        <a href="https://tinyurl.com/grass-rc2">tinyurl.com/grass-rc2</a> (run in Binder)
+    </p>
+    <p>by Vaclav Petras and Anna Petrasova</p>
+</section>
+
+
+<section>
+    <h2>Integration with Jupyter Notebooks</h2>
+    <p>
+        Time-series visualization
+    </p>
+    <img src="img/jupyter_timeseries_viewshed.gif" class="stretch">
+    <p>by Vaclav Petras and Anna Petrasova</p>
```

Comment:
Add Caitlin as well, as this was the topic of her student grant

Reply:
Sure, done in a61338f20952977bc9f9ca41a78e78390a419b73